docs/how-to/installation/packages.txt

```diff
@@ -5,14 +5,76 @@ Install Determined Using Debian Packages
 
 For systems running Ubuntu 16.04 or 18.04, we support installing the
 Determined master and agent using Debian packages and running them as
-systemd services rather than Docker containers.
-
-For PostgreSQL, you should use either your Linux distribution's package
-and service or a :ref:`Docker container <install-using-docker>`.
+systemd services.
 
 Preliminary Setup
 -----------------
 
+PostgreSQL
+~~~~~~~~~~
+Determined leverages a PostgreSQL database to store experiment and trial
+metadata. You may either leverage your Linux distribution's package and service
```

Comment:
Use some other word (perhaps "use") instead of "leverage" one of these two times. Maybe both.

Reply:
Done (changed both)